src/main/resources/META-INF/plugin.xml

```diff
@@ -211,6 +211,21 @@
             <!-- In $default keymap `control` key (Windows/Linux) corresponds to `cmd` key on macOS -->
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
         </action>
+
+        <action id="GitMachete.NextBranchCheckoutAction"
+                class="com.virtuslab.gitmachete.frontend.actions.traverse.NextBranchCheckoutAction">
+            <keyboard-shortcut keymap="$default" first-keystroke="control alt shift G" />
+        </action>
+
+        <action id="GitMachete.PreviousBranchCheckoutAction"
+                class="com.virtuslab.gitmachete.frontend.actions.traverse.PreviousBranchCheckoutAction">
+            <keyboard-shortcut keymap="$default" first-keystroke="control alt shift B" />
+        </action>
+
+        <action id="GitMachete.ParentBranchCheckoutAction"
+                class="com.virtuslab.gitmachete.frontend.actions.traverse.ParentBranchCheckoutAction">
+            <keyboard-shortcut keymap="$default" first-keystroke="control alt shift U" />
+        </action>
```

Comment:
Meanwhile, can you please do me a favour by trying these key combinations on Linux or Windows?

Reply:
Lemme check, as I'm probably the sole non-Mac (Linux) user around 😅

Reply:
They work like a charm 🥰 still, TBD what shortcuts to use... actually, how about the F<number> buttons? they seem extremely underused in IntelliJ...  other than the standard Alt+F4, F1 some F7-F10 for build/run/debug, and maybe Alt+F12 for listing class members, I can't think of any

Reply:
In Mac, the function keys do a lot of code navigation. Option + Function keys types strange characters.

Reply:
In Mac, Option is Alt

Reply:
@mkondratek any ideas here?

Reply:
what's wrong with arrow keys?

Reply:
they navigate in the text instead, even with ctrl+alt+shift

Reply:
well yes, but no

Reply:
I think we are overthinking this one. let's go for a simple **and known** solution - the arrow. 

They navigate through the editor - that's true unless the focus is elsewhere. 
It can be in Project overview - where we can navigate b/w files.
It can be in Git Log - which is almost exactly the application that we need. 
Try arrow navigation in Git Log for yourself - it's extremely easy and clear.

Ofc we will lose the potential possibility to navigate branches from anywhere (probably without git machete tab opened) but it makes things way more complicated and less readable. Let's stick to good and simple solution.

Reply:
for me, the more complicated key combination is, the less likely I will use it. 

I cannot remember using the shortcut for opening git machete tab for purposes other than testing this shortcut - this key, that key, stand on one leg, touch your nose, use different nose on Mac... who remembers that? 😆

Reply:
> Ofc we will lose the potential possibility to navigate branches from anywhere (probably without git machete tab opened) but it makes things way more complicated and less readable

Wise advice indeed, +1

Just some research is needed if it's easy/doable at all to make Git Machete tab capture the arrows  🤔 

> I cannot remember using the shortcut for opening git machete tab for purposes other than testing this shortcut - this key, that key, stand on one leg, touch your nose, use a different nose on Mac... who remembers that?

I do, all the time :trollface: but I'm keyboard shortcut addict basically (I've even set up a shortcut to install a plugin from disk ;p)

Reply:
well, I was wondering if someone has git-machete tab open, wouldn't it be much handier to just click on the needed branch for checking out?

Reply:
In general: yes, that's long implemented (double click checks out the branch).
ut
The aim of this entire task however, is to provide a gradual base for traverse-like feature: not sure if the (semi)-automatic traversal can reasonably be implemented in this plugin, and how it's gonna look like... but providing a way to jump b/w branches without clicking looks like a first good step to figuring out the best UX for traversal.

Hence, I'm willing to approve this PR even if **there's no built-in way to access the checkout {next,prev,parent}** actions. It still has the research value for traverse. However, if an easy&natural way to trigger these actions can be figured on this PR (like arrows?)... then I'm more than happy to take it :)

Reply:
So let me see what I can do with the arrows 👍

Reply:
The alt+up, alt+right, alt+left arrows are working for me on MacOS. Can you please do me a favour by trying them on a different operating system?

Reply:
I mean `Option`.
Now if the machete tab is open and on focus, the arrow keys work for branch navigation.

Reply:
Lemme check on Linux; Mikołaj pls double-verify on MacOS

Reply:
(works on Linux)